.github/CHANGELOG.md

```diff
@@ -527,6 +502,15 @@ random_mat2 = rng.standard_normal(3, requires_grad=False)
   have decompositions over elementary gates.
   [(#1278)](https://github.com/PennyLaneAI/pennylane/pull/1278)
 
+* The `qml.SWAP` operation now has a decomposition over elementary gates.
+  [(#1329)](https://github.com/PennyLaneAI/pennylane/pull/1329)
+
+* The `qml.Toffoli` operation now has a decomposition over elementary gates.
+  [(#1320)](https://github.com/PennyLaneAI/pennylane/pull/1320)
+
+* Added CPhase operation as an alias for ControlledPhaseShift operation
+  [(#1319)](https://github.com/PennyLaneAI/pennylane/pull/1319).
```

Comment:
Let's combine this, as well as the entry for

> A decomposition has been added for the qml.CSWAP operation.

into a single entry with nested bullet points, like you did above for

> * The following new operations have been added:

:)

Reply:
that is,

```markdown
* Decompositions in terms of elementary gates has been added for:

  - `qml.Toffoli` [(#1320)](https://github.com/PennyLaneAI/pennylane/pull/1320)